### PR TITLE
Updated node engine to 10.15.1

### PR DESCRIPTION
**Pull request by jordanrendall**
Npm was not able to install packages on heroku using node v4.4.5. Heroku recommended upgrading to at least 4.7.0.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Hello~!

We are currently discussing removing the `engine` field from the `package.json` entirely: https://github.com/freeCodeCamp/freeCodeCamp/issues/40365

Would you like to update this PR to make that change?

File: package.json
Comment (moT01) [inline package.json:13]:
```suggestion
```

File: package.json
Comment (moT01) [inline package.json:15]:
```suggestion
```

File: package.json
Comment (moT01) [inline package.json:14]:
```suggestion
```

**Review by moT01 — COMMENTED** [review]
I have made the suggested suggestions 😄

**Comment by moT01** [comment]
Want to give this a quick review @nhcarrigan || @ShaunSHamilton

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳

Please note that our workflow requires at least one additional approval before this can be merged. 🙂

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM :thumbsup:

**Comment by ShaunSHamilton** [comment]
Congrats on your first PR to this repo. We look forward to your future contributions.

**Pull request merged**
